packages/web/package.json

```diff
@@ -28,6 +26,10 @@
     "@types/jest": "^23.3.13",
     "@types/react": "^16.7.22",
     "@types/react-dom": "^16.0.11",
+    "@types/react-relay": "1.3.10",
+    "@types/rebass": "0.3.4",
+    "@types/styled-components": "4.1.6",
+    "@types/styled-system": "3.1.3",
```

Comment:
I think we can move all @types to root package

Reply:
Even if types are not "common"?

Reply:
let them here them